# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../mythtv-plugins/gcc41.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur mythplugins-0.19-orig/mythgame/mythgame/gamehandler.h mythplugins-0.19/mythgame/mythgame/gamehandler.h
--- mythplugins-0.19-orig/mythgame/mythgame/gamehandler.h	2005-11-10 03:54:23.000000000 +0000
+++ mythplugins-0.19/mythgame/mythgame/gamehandler.h	2006-04-02 13:17:49.086989750 +0000
@@ -106,7 +106,7 @@
 
   protected:
     static GameHandler* GetHandler(RomInfo *rominfo);
-    static GameHandler* GameHandler::GetHandlerByName(QString systemname);
+    static GameHandler* GetHandlerByName(QString systemname);
 
     bool rebuild;
     int spandisks;
diff -Nur mythplugins-0.19-orig/mythphone/mythphone/sipfsm.h mythplugins-0.19/mythphone/mythphone/sipfsm.h
--- mythplugins-0.19-orig/mythphone/mythphone/sipfsm.h	2005-04-05 22:15:30.000000000 +0000
+++ mythplugins-0.19/mythphone/mythphone/sipfsm.h	2006-04-02 13:57:23.967410500 +0000
@@ -618,7 +618,7 @@
     SipWatcher *CreateWatcherFsm(QString Url);
     SipIM *CreateIMFsm(QString Url="", QString callIdStr="");
     void StopWatchers();
-    void SipFsm::KickWatcher(SipUrl *Url);
+    void KickWatcher(SipUrl *Url);
     void SendIM(QString destUrl, QString CallId, QString imMsg);
     int numCalls();
     int getPrimaryCall() { return primaryCall; };
diff -Nur mythplugins-0.19-orig/mythphone/mythphone/vxml.h mythplugins-0.19/mythphone/mythphone/vxml.h
--- mythplugins-0.19-orig/mythphone/mythphone/vxml.h	2005-02-08 09:34:05.000000000 +0000
+++ mythplugins-0.19/mythphone/mythphone/vxml.h	2006-04-02 13:57:06.090293250 +0000
@@ -10,7 +10,7 @@
 {
   public:
     vxmlVariable(QString N, QString V);             
-    vxmlVariable::vxmlVariable(QString N, short *wav, int S);
+    vxmlVariable(QString N, short *wav, int S);
     virtual ~vxmlVariable() {}; 
     bool isType(QString t) { return (t == Type); };
     QString getName() { return Name; };
diff -Nur mythplugins-0.19-orig/mythvideo/mythvideo/videomanager.h mythplugins-0.19/mythvideo/mythvideo/videomanager.h
--- mythplugins-0.19-orig/mythvideo/mythvideo/videomanager.h	2005-10-04 15:48:34.000000000 +0000
+++ mythplugins-0.19/mythvideo/mythvideo/videomanager.h	2006-04-02 14:02:15.205611750 +0000
@@ -30,7 +30,7 @@
   public:
     VideoManager(MythMainWindow *parent, const char *name = 0);
     ~VideoManager(void);
-    void VideoManager::processEvents() { qApp->processEvents(); }
+    void processEvents() { qApp->processEvents(); }
     
   public slots:
     void slotManualIMDB();
diff -Nur mythplugins-0.19-orig/mythvideo/mythvideo/videoselected.h mythplugins-0.19/mythvideo/mythvideo/videoselected.h
--- mythplugins-0.19-orig/mythvideo/mythvideo/videoselected.h	2005-07-12 18:21:05.000000000 +0000
+++ mythplugins-0.19/mythvideo/mythvideo/videoselected.h	2006-04-02 14:06:42.310304750 +0000
@@ -35,7 +35,7 @@
     VideoSelected(VideoList *lvideolist,
                  MythMainWindow *parent, const char *name = 0, int index = 0);
     ~VideoSelected();
-    void VideoSelected::processEvents() { qApp->processEvents(); }
+    void processEvents() { qApp->processEvents(); }
     
 
   protected slots:
